symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/BotAuthenticator.java

```diff
@@ -16,4 +18,18 @@
    * @return the authentication session.
    */
   @Nonnull AuthSession authenticateBot();
+
+  /**
+   *
+   * @return a new session token
+   * @throws AuthUnauthorizedException in case of authentication failure
+   */
+  @NonNull String retrieveSessionToken() throws AuthUnauthorizedException;
+
+  /**
+   *
+   * @return a new key manager token
+   * @throws AuthUnauthorizedException in case of authentication failure
+   */
+  @NonNull String retrieveKeyManagerToken() throws AuthUnauthorizedException;
```

Comment:
The initial idea to retrieve `sessionToken` and `keyManagerToken` was actually to use the `AuthSession#getSessionToken()` and `AuthSession#getKeyManagerToken()`. Adding those methods here is redundant.